app/controllers/openid_connect/authorization_controller.rb

```diff
@@ -79,11 +76,13 @@ def handle_successful_handoff
       delete_branded_experience
     end
 
-    def track_authorize_analytics(result, extra = {})
-      extra[:user_fully_authenticated] = user_fully_authenticated?
-      analytics_attributes = result.to_h.except(:redirect_uri).merge(extra)
+    def track_handoff_analytics(result, attributes = {})
+      result_attributes = result.to_h
+      attributes[:success] = result.success?
+      attributes[:client_id] = result_attributes[:client_id]
+      attributes[:code_digest] = result_attributes[:code_digest]
 
-      analytics.openid_connect_request_authorization(**analytics_attributes)
+      analytics.openid_connect_authorization_handoff(**attributes)
```

Comment:
ditto about inlining maybe? also doesn't modify the passed-in param
```ruby
      analytics.openid_connect_authorization_handoff(
        **attributes.merge(result.to_h.slice(:client_id, :code_digest)).merge(
          success: result.success?,
        ),
      )
```

Reply:
Good call, changed in fa48132086